### PR TITLE
changes

### PR DESCRIPTION
**Pull request by Tanuja0077**

### CODE REVIEW

File: src/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './redux/Todo.css'
+import TodoItem from './TodoItem'
+import { useSelector } from 'react-redux'
+
+const TodoList = () => {
+     const todos = useSelector(state => state);
+    return (
```
Comment (endouAxel) [inline src/TodoList.js:8]: here it should be state => state.yourVariableInStore (state.todos or state.newTodos) based on how you are storing.

Comment (endouAxel) [reply src/TodoList.js:8]: to do that you need to return an object which is proper way.

File: src/redux/TodoReducers.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                if (newTodos[i].id === action.payload.id) {
+                    break;
+                }
+
+            }
+            if (index !== -1) {
+                newTodos[index] = action.payload;
+                return newTodos;
+            }
+
```
Comment (endouAxel) [inline src/redux/TodoReducers.js]: In case of Update Item you are not returning anything

Comment (endouAxel) [reply src/redux/TodoReducers.js]: which is undefined (undefined.map is not a function)

**Review by endouAxel — CHANGES_REQUESTED** [review]

File: src/TodoList.js (hunk 1 above)
Comment (endouAxel) [reply src/TodoList.js:8]: Here you are accessing in proper way. but in reducer for update item you are returning undefined

**Comment by Tanuja0077** [comment]
If statement lo return chesa

File: src/TodoList.js (hunk 1 above)
Comment (endouAxel) [inline src/TodoList.js:8]: it should be useSelector(state => state. TodoReducer)

**Pull request merged**